src/main/java/com/bhoruka/bloodbank/model/request/SearchCampRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.bhoruka.bloodbank.model.request;
+
+import java.util.Date;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class SearchCampRequest {
```

Comment:
Are we going to handle AND and OR searches? For example, 
1. Search for all camps where partnerId = A OR partnerId = B
2. Search for all camps where partnerId = A AND dateOfCamp between(date1, date2)